types/babel__traverse/index.d.ts

```diff
@@ -90,7 +90,13 @@ export class Scope {
 
     removeData(key: string): void;
 
-    push(opts: any): void;
+    push(opts: {
+        id: object,
+        init?: object,
+        unique?: boolean,
+        _blockHoist?: number,
```

Comment:
Is this a supported API or for internal Babel use only?

Reply:
Presumably internal - I can't think of another reason why an underscore would be used here.